src/planwise/client/current_project/db.cljs

```diff
@@ -58,21 +65,33 @@
                         :request nil
                         :request-with nil}
 
+   :view-state         :project                   ; [:project :share-dialog]
+
+   :shares             (asdf/new [])              ; [ProjectShare]
+   :sharing            {:emails-text nil
+                        :token-state nil          ; [nil :reloading]
```

Comment:
Why not use `asdf` here? It does require `dissoc`ing the token before updating `project-data` (to avoid duplication), but since it's a special piece of info that can be updated individually it makes sense. Otherwise, use an `asdf` inside `project-data`. In that case updates to `project-data` need to be controlled but I think it needs to change in `db/update-viewmodel` only.